SPECS/systemd/50-security-hardening.conf

```diff
@@ -17,3 +17,6 @@ net.ipv4.ip_forward = 0
 net.ipv6.conf.all.forwarding = 0
 #Disable IPv6 Router Advertisements
 net.ipv6.conf.all.accept_ra = 0
+# Security patch for msid: 1.5.1
+# Ensure core dumps are restricted
+fs.suid_dumpable = 0
```

Comment:
Talked with Chaitanya, we really want these on for most images to help us debug customer issues. This should be a Fedramp only flag.